plugins/api-docs/src/components/AsyncApiDefinitionWidget/AsyncApiDefinition.tsx

```diff
@@ -142,6 +142,28 @@ const useStyles = makeStyles((theme: BackstageTheme) => ({
   },
 }));
 
+const fetchResolver = {
+  order: 199,
+  canReadValue: '^https?://[^\\s$.?#].[^\\s]*$',
```

Comment:
Quick elaboration on these two could be useful, in a comment or such. Why is the order `199`? :grin: and perhaps it's enough that the regex only checks for `^https?://`? I don't know if it is, but could use some motivation otherwise

Reply:
Changed the regex. 199 is required to execute before the built-in resolver https://github.com/APIDevTools/json-schema-ref-parser/blob/main/lib/resolvers/http.js#L15

Reply:
I know why the order is 199, but not everyone that reads this code might immediately understand it :grin: It's a bit of a magic number, so suggesting we throw a comment on it to make it clear that this places us just before the built-in resolver

Reply:
ahh got you 😉